### PR TITLE
[TensorV2] Add subtraction operation structure

### PR DESCRIPTION
**Pull request by djeong20**
This pull request introduces a basic structure of tensor subtraction operations.
Function implementation will be added in a later PR.

** Self-evaluation**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test:   [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: nntrainer/tensor/tensor_v2.h [hunk 1]
```diff
@@ -382,8 +382,8 @@ class TensorV2 {
                 float value, float beta) noexcept;
 
   /**
-   * @brief 
-   * 
+   * @brief
```
Comment (SeoHyungjun) [inline nntrainer/tensor/tensor_v2.h:385]: There is no brief.
Aren't you trying to include a brief about setZero?

**Review by SeoHyungjun — APPROVED** [review]
LGTM

File: nntrainer/tensor/tensor_v2.h (hunk 1 above)
Comment (djeong20) [reply nntrainer/tensor/tensor_v2.h:385]: it was accidentally removed during the rebase. added back in #2439!

**Pull request merged**